src/common/sha3/ossl_sha3x4.c

```diff
@@ -81,7 +81,10 @@ static void SHA3_shake128_x4_inc_squeeze(uint8_t *out0, uint8_t *out1, uint8_t *
 		OSSL_FUNC(EVP_MD_CTX_copy_ex)(clone, s->mdctx3);
 		OSSL_FUNC(EVP_DigestFinalXOF)(clone, out3, outlen);
 	} else {
-		uint8_t *tmp = OQS_MEM_checked_malloc(s->n_out + outlen);
+		uint8_t *tmp = OQS_MEM_malloc(s->n_out + outlen);
+		if (!tmp) {
+			return;
```

Comment:
This and similar checks should (for now, at least) call `abort` or `exit`—the SHA3 init functions are not checked in the code that uses them (maybe they should be, but that's out of scope here IMO).

Reply:
@SWilson4 For the long term, we should return error codes and allow the caller to determine the appropriate behavior. Can we, for now, use return instead of invoking abort() or exit()?

We shouldn't assume that the caller process needs to terminate if SHA3_shake128_x4_inc_init fails. Otherwise, we won’t be able to integrate liboqs into our system. For example, in some cases, SHA3_shake128_x4_inc_init is not a critical function, and the caller may choose to continue the process even if it fails.

Please let me know your thoughts.